spec/views/category/index.html.erb_spec.rb

```diff
@@ -10,19 +10,10 @@
     end
     before do
       visit root_path
-
-      # Click the 'Sign in' link
-      click_link 'Sign In'
-
-      fill_in 'Email', with: user.email
-      fill_in 'Password', with: user.password
-
-      # Click the sign-in button (adjust the selector based on your UI)
-      click_button 'Log in'
     end
 
     it 'renders the index view' do
-      expect(page).to have_content('ADD CATEGORY')
+      expect(page).to have_content('Budget')
     end
```

Comment:
- [ ] Hi, Great job for trying to fix the test. I would suggest you try again because the test is still falling.

- Test Failed
![Capture](https://github.com/danielochuba/budget-app/assets/63972670/a38311f3-bd1c-4995-9eaf-80f8bf08d951)